geometry_msgs/Pose obj_pose
geometry_msgs/Pose navigation_pose
string color
uint8 id